### PR TITLE
Light wallet linux install guide

### PR DESCRIPTION
**Pull request by ohcee**
Light-wallet LInux install guide, light-wallet photos, and updated CSS.

### CODE REVIEW

File: Veil-light-wallet-Linux.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+            <figure>
+                <img src="light-wallet-pictures/veil_wallet_gear_icon.png" alt="veil_wallet_gear_icon">
+                <figcaption>Veil-wallet gear icon.</figcaption>
+            </figure>
+        <li>This should successfully launch your Veil Light Wallet.</li>
+        <li>Configure any initial settings, such as language.</li>
+        <li>Ensure you record and securely store your seed phrase generated during the setup. You can also enter a seed phrase from a previous Veil wallet.</li>
+        <li>After entering your wallet phrase correctly, let your wallet load.</li>
+        <li>The wallet should be ready to use in a few seconds up to a half-hour.</li>
+        <li>Note that since Veil is private by default, this light wallet uses RingCT, and basecoin will not be usable, visible, or available with this light wallet.</li>
```
Comment (steel97) [inline Veil-light-wallet-Linux.html:73]: Same as for iphone guide (CT is also currently unavailable)

File: Veil-light-wallet-Linux.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            <img src="light-wallet-pictures/new_folder_w_files.png" alt="new_folder_w_files">
+            <figcaption>New folder on top.<br>
+            Downloaded wallet files on bottom.</figcaption>
+        </figure>
+        <li>Open the terminal and navigate to your wallet location. If you have the folder on your desktop, <br>
+            type <code>cd Desktop</code> and then <code>cd veil\ light\ wallet/</code> (or whatever you named the folder).</li>
+            <figure>
+                <img src="light-wallet-pictures/light_wallet_example2.png" alt="light_wallet_example">
+            </figure>
+        <li>Once in the folder, type <code>chmod +x veil-wallet</code> to make the veil-wallet file executable.</li>
```
Comment (steel97) [inline Veil-light-wallet-Linux.html:50]: Would be good to add information about changing permissions without terminal (in ubuntu it's possible in file manager by right-clicking on executable `Properties > Allow executing as program`. I had this option in 23.10, but it also should be available in 22.04)

**Comment by seanPhill** [comment]
Also some other observations 

1. When unzipped on my Ubuntu notebook, the zip file is already a zip of a folder, so I had to move the extracted folder up `cd ../..` into the veil_lightwallet folder I had extracted it into.
2. When importing an existing veil wallet seed I had to scroll slightly, actually [tab] worked, to reveal the whole of the action button to create the wallet from the seed I had typed. Fortunately some of the button was visible, making it apparent to the user that slightly scrolling was necessary.
3. I have a more than ten years old laptop running Ubuntu, so it doesn't have any biometrics, so I selected [Skip (not recommended)] and then it instantly said "Failed to connect to node. Please try again. OK", so it seems that unlike when I did it on the Android version the default node is not set or running? Trying again failed and I couldn't find any way to select a node. Clicking back led me to the seed phrase page again.
4. Clicking [Advanced] leads to the form to set an encryption password (also available via the first visible button in the seed phrase importing screen). I don't see any node settings.
5. I have to go back to the "Welcome to Veil!" screen to find the [Settings] button. MAYBE people should be directed to the Settings button before doing anything else, and some guidance (in this guide) may be advisable on the purpose and meaning of the options? … Nicely, the gears icon next to the "Node url" setting gives the choice of the US and EU nodes, but having checked that it is running on my Android phone I would have expected it not to have failed at point 3, "Failed to connect to node." As I'm going to have to type the whole seed phrase again to try again, I'll try the EU node this time, and switch the language.

… It worked when I set a passphrase and had set the EU node, but **maybe** I made a mistake with the seed phrase (or recorded it wrong when I did the Android), as I got a zero balance, no transactions listed, and a different address.

After these two run throughs, both wallets were visible in the wallet selection list, as I gave them different names, so the first time through did work, except that maybe it was only a momentary failure to reach the node on this computer, that hadn't fetched an IP address for the node before.

**Comment by steel97** [comment]
> 3\. I have a more than ten years old MacBook Pro running Ubuntu, so it doesn't have any biometrics, so I selected [Skip (not recommended)] and then it instantly said "Failed to connect to node. Please try again. OK", so it seems that unlike when I did it on the Android version the default node is not set or running? Trying again failed and I couldn't find any way to select a node. Clicking back lead me to the seed phrase page again.

Node connection fixed in latest builds from actions tab (biometrics buttons also completely removed on linux client)

**Comment by steel97** [comment]
This guide is suitable for upcoming release (1.0.7, which I try to release before end of 2023)

File: Veil-light-wallet-Linux.html (hunk 1 above)
Comment (ohcee) [reply Veil-light-wallet-Linux.html:73]: Will add that CT and basecoin are unavailable.

File: Veil-light-wallet-Linux.html (hunk 2 above)
Comment (ohcee) [reply Veil-light-wallet-Linux.html:50]: Totally forgot about that way. Will add with pictures.

**Comment by ohcee** [comment]
> Also some other observations
> 
>     1. When unzipped on my Ubuntu notebook, the zip file is already a zip of a folder, so I had to move the extracted folder up `cd ../..` into the veil_lightwallet folder I had extracted it into.
> 
>     2. When importing an existing veil wallet seed I had to scroll slightly, actually [tab] worked, to reveal the whole of the action button to create the wallet from the seed I had typed. Fortunately some of the button was visible, making it apparent to the user that slightly scrolling was necessary.
> 
>     3. I have a more than ten years old laptop running Ubuntu, so it doesn't have any biometrics, so I selected [Skip (not recommended)] and then it instantly said "Failed to connect to node. Please try again. OK", so it seems that unlike when I did it on the Android version the default node is not set or running? Trying again failed and I couldn't find any way to select a node. Clicking back led me to the seed phrase page again.
> 
>     4. Clicking [Advanced] leads to the form to set an encryption password (also available via the first visible button in the seed phrase importing screen). I don't see any node settings.
> 
>     5. I have to go back to the "Welcome to Veil!" screen to find the [Settings] button. MAYBE people should be directed to the Settings button before doing anything else, and some guidance (in this guide) may be advisable on the purpose and meaning of the options? … Nicely, the gears icon next to the "Node url" setting gives the choice of the US and EU nodes, but having checked that it is running on my Android phone I would have expected it not to have failed at point 3, "Failed to connect to node." As I'm going to have to type the whole seed phrase again to try again, I'll try the EU node this time, and switch the language.
> 
> 
> … It worked when I set a passphrase and had set the EU node, but **maybe** I made a mistake with the seed phrase (or recorded it wrong when I did the Android), as I got a zero balance, no transactions listed, and a different address.
> 
> After these two run throughs, both wallets were visible in the wallet selection list, as I gave them different names, so the first time through did work, except that maybe it was only a momentary failure to reach the node on this computer, that hadn't fetched an IP address for the node before.

@seanPhill can you try the updated version in the actions and tell me if you have the same issues. The guide is aimed at version 1.0.7. That will now be mentioned in the updated guide. I will also update instructions using the release file not the development version.

**Review by steel97 — APPROVED** [review]

**Pull request merged**